index.html

```diff
@@ -4,10 +4,197 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>CSS Frameworks CA</title>
+    <meta
+    name="description"
+    content="SoMe - Social media app for"
+  />
+    <title>SoMe One - CSS Frameworks CA</title>
     <link rel="stylesheet" href="/dist/css/styles.css" />
   </head>
   <body>
-    <h1>CSS Frameworks CA</h1>
+    <header>
+      <nav
+        class="navbar navbar-expand-lg navbar-dark bg-darkorange"
+        aria-label="Eighth navbar example"
+      >
+        <div class="container">
+          <a class="navbar-brand" href="index.html">SoMe One</a>
+          <button
+            class="navbar-toggler bg-darkgrey"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarsExample07"
+            aria-controls="navbarsExample07"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navbarsExample07">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="profile.html">Profile</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="login.html">Login</a>
+              </li>
+            </ul>
+            <form role="search">
+              <input
+                class="form-control"
+                type="search"
+                placeholder="Search"
+                aria-label="Search"
+              />
+            </form>
```

Comment:
Would be great to have a submit button, for example with a magnifying glass icon on it. Unless you're wanting results to show up as a dropdown for matching elements, such as with a typeahead search?

Reply:
Thanks for the tips. An dropdown search would have been nice actually. But for now I just added an search button with an magnifying glass icon!